scipy/signal/filter_design.py

```diff
@@ -4863,6 +4865,201 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
```

Comment:
Is there a reason this has to be private?  It is a useful function in its own right.  I know it is simple, but if it is going to be in there, I think it might as well be available for use.